src/gpuarray/buffer.h

```diff
@@ -490,11 +490,14 @@ GPUARRAY_PUBLIC int gpukernel_binary(gpukernel *k, size_t *sz, void **obj);
 GPUARRAY_PUBLIC int gpukernel_property(gpukernel *k, int prop_id, void *res);
 
 GPUARRAY_PUBLIC gpucontext *gpukernel_context(gpukernel *k);
+
 /**
  * \defgroup props Properties
  * @{
  */
 /* Start at 1 for GA_CTX_PROP_ */
+#define GA_CTX_PROP_START    1
```

Comment:
yeap sorry, I saw what you wrote about magic numbers and I added 3 more macros to define positions of CTX, BUFFER, KERNEL in `Properties` defgroup